frontend/src/cards/UnknownsMapCard.tsx

```diff
@@ -58,11 +58,9 @@ function UnknownsMapCardWithKey(props: UnknownsMapCardProps) {
   const preloadHeight = useGuessPreloadHeight([700, 1000]);
   const metricConfig = props.variableConfig.metrics["pct_share"];
   const currentBreakdown = props.currentBreakdown;
-  const breakdownString =
-    BREAKDOWN_VAR_DISPLAY_NAMES_LOWER_CASE[currentBreakdown];
-
+  const breakdownString = `with unknown ${BREAKDOWN_VAR_DISPLAY_NAMES_LOWER_CASE[currentBreakdown]}`;
```

Comment:
this creates a bug `We do not currently have Share of all voter participation **broken down by** with unknown`

 (i think it's from this line)... 

<img width="1268" alt="Screen Shot 2022-12-09 at 11 37 34 AM" src="https://user-images.githubusercontent.com/41567007/206770143-e4e188a4-afd7-4cd2-954d-2e422c28dc1c.png">